modules/kubernetes-addons/aws-load-balancer-controller/locals.tf

```diff
@@ -4,13 +4,14 @@ locals {
 
```

Comment:
the only change we need to make is on line 3

```hcl
service_account_name = try(var.helm_config.service_account_name, "${local.name}-sa")
```

Reply:
@bryantbiggs addressed. Thank you for the review.